src/acquisition/flusurv/flusurv_update.py

```diff
@@ -20,25 +20,40 @@
 =======================
 
 `flusurv` is the table where US flu hospitalization rates are stored.
-+--------------+-------------+------+-----+---------+----------------+
-| Field        | Type        | Null | Key | Default | Extra          |
-+--------------+-------------+------+-----+---------+----------------+
-| id           | int(11)     | NO   | PRI | NULL    | auto_increment |
-| release_date | date        | NO   | MUL | NULL    |                |
-| issue        | int(11)     | NO   | MUL | NULL    |                |
-| epiweek      | int(11)     | NO   | MUL | NULL    |                |
-| location     | varchar(32) | NO   | MUL | NULL    |                |
-| lag          | int(11)     | NO   | MUL | NULL    |                |
-| rate_age_0   | double      | YES  |     | NULL    |                |
-| rate_age_1   | double      | YES  |     | NULL    |                |
-| rate_age_2   | double      | YES  |     | NULL    |                |
-| rate_age_3   | double      | YES  |     | NULL    |                |
-| rate_age_4   | double      | YES  |     | NULL    |                |
-| rate_overall | double      | YES  |     | NULL    |                |
-| rate_age_5   | double      | YES  |     | NULL    |                |
-| rate_age_6   | double      | YES  |     | NULL    |                |
-| rate_age_7   | double      | YES  |     | NULL    |                |
-+--------------+-------------+------+-----+---------+----------------+
++-------------------+-------------+------+-----+---------+----------------+
+| Field             | Type        | Null | Key | Default | Extra          |
++-------------------+-------------+------+-----+---------+----------------+
```

Comment:
this textual "table" is more useful than its counterpart in the DDL file, though it suffers from the same pitfalls.  its good as a local reference for someone working on this file, but that also makes it dangerous if its not perfectly in sync and up-to-date.  id argue for removing it, but its ok to leave here too.

Reply:
I'll point to the file defining the schema instead